docs/api/time.rst

```diff
@@ -35,6 +35,10 @@ datatable.time
    * - :func:`ymd(y,m,d)`
      - Create a ``date32`` column from year, month, and day components.
 
+   * - :func:`ymdt(y,m,d,H,M,S)`
+     - Create a ``time64`` column from year, month, day, hour, minute, and
+       second components.
```

Comment:
Do we want to mention nanoseconds here?

Reply:
Nanoseconds is an optional parameter, that I don't expect to be used very often. Here in the overview we show the minimal signature of the function.